app/models/alchemy_solidus/touch_alchemy_ingredients.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module AlchemySolidus
+  module TouchAlchemyIngredients
+    extend ActiveSupport::Concern
+
+    included do
+      after_save :touch_alchemy_ingredients
```

Comment:
Would update work as well? New products, taxons, etc. will very unlikely already be assigned to an ingredient, no?